framework/logging.cpp

```diff
@@ -2217,6 +2222,17 @@ void YamlLogger::print(int, ChildExitStatus status)
     logging_printf(LOG_LEVEL_VERBOSE(1), "  test-runtime: %s\n",
                    format_duration(test_duration, FormatDurationOptions::WithoutUnit).c_str());
 
+    std::valarray<long double> freqs(num_cpus());
```

Comment:
In this case, primarily for the overloaded `sum()` method. To be fair, nothing I'm doing here *requires* it. If there's an aversion to using a `valarray`, I'm fine using a `vector`.

Reply:
I guess I was really wondering why use a collection at all instead of simply doing 

```
long double freqs = 0.0;
for (int i = 0; i < num_cpus(); i++)
  freqs += data->eff_freq_mhz;

const long double freq_avg = 1.0L * freqs / num_cpus();
```

Reply:
Tl;dr - You're right, I'll nix the container.

In an early iteration I did some additional statistics, and the valarray had some attractive conveniences. After I stripped it down to only the mean, the valarray stuck around, and I'd convinced myself the `sum()` method was oh-so-nice. In this instance, I missed the forest for the trees 🤣 .